matveev_alexey/task3.cpp

```diff
@@ -94,6 +95,12 @@ class Graph {
     throw std::runtime_error("Cannot be reached.");
   }
 
+  const std::vector<EdgeId> vertexConnections(VertexId id) const {
+    return connection_list_.at(id);
+  }
+  const std::vector<Vertex>& graphVertexes() const { return vertexes_; }
+  const std::vector<Edge>& graphEdges() const { return edges_; }
```

Comment:
Можно проще
```suggestion
  const std::vector<Vertex>& vertexes() const { return vertexes_; }
  const std::vector<Edge>& edges() const { return edges_; }
```
Что из этого, кажется тебе логичней?
```cpp
graph.graphVertexes();
graph.vertexes();
```